components/Buttons/MarkWatchedButton.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <v-btn v-if="canMarkWatched" icon dark @click.stop.prevent="togglePlayed">
+    <v-icon>
+      {{ isPlayed ? 'mdi-check-all' : 'mdi-check' }}
```

Comment:
I'd not use this icon in the `isPlayed===true` state. I'd go with the same thing as jf-web, changing its color. The all-checked icon looks like we selected some stuff.

Maybe another thing, why not use an eye icon for the watch status instead of the check? (just an idea though)

Reply:
I wasn't a fan of the colors before. I've updated it to show blue if it has been watched, else white